docs/tutorials/anomaly-detection.rst

```diff
@@ -0,0 +1,614 @@
+
+.. mermaid::
+
+    graph LR;
+
+        id1(IoT device)-- sensor reading -->id2(average 30 seconds);
+        id2-- average reading -->id4(over threshold?);
+        id4-- yes --> id5(notification);
+        id3(threshold table)-- thresholds --> id4;
+
+.. Note::
+
+    Even if the sample dataset and the data pipeline examples might seem basic, they offer a wide coverage on the integration and transformation options available with Apache Flink that can be applied to other, more complex, scenarios. 
+
+The tutorial includes:
+
+* Apache Flink for data transformation
+* Apache Kafka for data streaming
+* PostgreSQL® for data storage/query
+* Slack as notification system
+
+.. Tip::
+
+    All the tools listed above are fully open source, and can therefore be installed and used locally. You're going to use the comparable Aiven services to avoid any network and integration complexities. 
+
+Architecture overview
+---------------------
+
+In this tutorial you'll build a streaming data pipeline. The sources will be an Apache Kafka topic, containing the fake stream of IoT metrics data, and a PostgreSQL® database that containing the alerting thresholds. Then, an Apache Flink® service, will combine the data, apply some transformation SQL to find the anomalies, and push the result to a separate Apache Kafka® topic or a Slack channel for team notification.
+
+
+.. mermaid::
+
+    graph TD;
+
+        id1(Apache Kafka)-- IoT metrics stream -->id3(Apache Flink);
+        id2(PostgreSQL)-- alerting threshold data -->id3;
+        id3-- filtered/aggregated data -->id1;
+        id3-- filtered data -->id7(Slack);
+
+Prerequisites
+-------------
+
+The tutorial uses Aiven services, therefore you'll need a valid `Aiven account <https://console.aiven.io/signup>`_. On top of the Aiven account, you will also need the following three items:
+
+* **Docker**, needed for the `fake data generator for Apache Kafka <https://github.com/aiven/fake-data-producer-for-apache-kafka-docker>`_. Check out the `related installation instructions <https://docs.docker.com/engine/install/>`_.
+* **Slack App and Token**: the data pipeline output is a notifications to a Slack channel, check out the needed steps to `set up a Slack app and retrieve the Slack authentication token <https://github.com/aiven/slack-connector-for-apache-flink>`_.
```

Comment:
The only part that made me stuck is understanding Slack connector :D and the info in github isn't too helpful for those with 0 knowledge.... some more hints would be highly appreciated :)

Reply:
I believe this will make the tutorial even longer. I tried making the instruction in GH a bit easier to follow with a PR recently, since most of the work needs to be performed in Slack, I would leave this out of scope for the tutorial